scripts/install-chef-suse.sh

```diff
@@ -278,6 +278,15 @@ post_fail_handler ()
     rm -f $crowbar_install_dir/crowbar_installing
 }
 
+function rails_db_cleanup()
+{
+    rm -f $crowbar_install_dir/crowbar-install-failed
+    rm -f $installation_steps
+    pushd /opt/dell/crowbar_framework > /dev/null
+    rake db:migrate:reset
```

Comment:
As mentioned in the previous pull request we can migrate down to `sessions` table to be keep the session data working. An example will follow.